announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm

```diff
@@ -345,6 +346,8 @@
 					$tlang.getString("gen.therearepublic")
 				#elseif ($!view == "view.bygroup")
 					$tlang.getString("gen.therearegroup")
+				#elseif ($!view == "view.byrole")
+					$tlang.getString("gen.therearegroup")
```

Comment:
@mpellicer Shouldn't this line be $tlang.getString("gen.therearerole") ?

Reply:
True! Thanks for the review!

I don't see where is happening, probably not visible, but you're right!

![image](https://user-images.githubusercontent.com/8653754/221167564-48f13ec4-442d-4447-ba0e-63aa34ff3b28.png)

![image](https://user-images.githubusercontent.com/8653754/221167591-e677f647-140b-4175-bbf3-037a5626c160.png)


M

Reply:
@susowrc thanks again and good catch, this commit fixes the issue!

https://github.com/sakaiproject/sakai/commit/26b24bd010b053c0b5d185db5388d18c4ad5f26d

Reply:
you’re welcome!